c++/src/kj/test.h

```diff
@@ -128,6 +141,17 @@ bool expectFatalThrow(Maybe<Exception::Type> type, Maybe<StringPtr> message,
 // fork() is not available, this always returns true.
 #endif
 
+bool expectExit(Maybe<int> statusCode, Function<void()> code);
```

Comment:
You can use `FunctionParam` instead of `Function` since the function will always be called synchronously.